styles/globals.css

```diff
@@ -11,9 +11,13 @@ a {
   text-decoration: none;
 }
 
-
-
-
 * {
   box-sizing: border-box;
 }
+:root {
+  --main-color: #f0faff;
+  --dark-green: #7e9aa3;
```

Comment:
Is it green? Or is it blue? 😅

Reply:
Hehe, had to look it up, and it contains more blue than green in the rgb-code ([here](https://www.color-hex.com/color/7e9aa3)). However, compared to the background it is more green.. So to differentiate, it could be nice to call it green. What do you think?

Reply:
ur mama said blue, than blue it izzz 💙